app/pages/sections/order-items/catalogue-solutions/delete/controller.js

```diff
@@ -1,5 +1,26 @@
+import { getContext } from './contextCreator';
+import { getOrderDescription } from '../../../../../helpers/routes/getOrderDescription';
 import { deleteCatalogueSolution as apiDeleteCatalogueSolution } from '../../../../../helpers/api/ordapi/deleteCatalogueSolution';
 
+export const getDeleteCatalogueSolutionContext = async ({
+  req,
+  sessionManager,
+  accessToken,
+  logger,
+}) => {
+  const { orderId } = req.params;
+  const { orderItemId } = req.params;
+  const { solutionName } = req.params;
+  const orderDescription = await getOrderDescription({
+    req,
+    sessionManager,
+    accessToken,
+    logger,
+  });
+
+  return getContext({ orderId, orderItemId, solutionName, orderDescription });
```

Comment:
It looks like there may be a couple of linting issues here:

```shell
app/pages/sections/order-items/catalogue-solutions/delete/controller.js
  21:21  error  Expected a line break after this opening brace   object-curly-newline
  21:76  error  Expected a line break before this closing brace  object-curly-newline
```

Reply:
resolved